src/main/react/src/views/AddRecipe.js

```diff
@@ -60,34 +64,49 @@ const AddRecipe = () => {
     getTags().then(data => setAllTags(data));
   }, []);
 
-  const postRecipe = () => {
+  const postRecipe = async () => {
+    if (file === null) {
+      setErrMsg("Upload a picture of your recipe!");
+      return;
+    } else if (title === "Title" || title === "") {
```

Comment:
Does React provide some nicer input validation lib? I was able sort of get around the non empty requirement:
http://meltingpot-step-2020.appspot.com/#/recipe?id=baIGuiElCA99HTe18u0G

Reply:
We can change it to a regex maybe? Honestly, if users want to get around validation, they can probably do that regardless of what validation we put in.

Reply:
I don't think regex is worth. I know bootstrap has built in validators. Looks like you'd need to install some third party npm for react. I don't think it is necessary.